src/components/fidelity_bond/FidelityBondDetailsSetupForm.tsx

```diff
@@ -0,0 +1,582 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react'
+import { Link } from 'react-router-dom'
+import * as rb from 'react-bootstrap'
+import { useTranslation } from 'react-i18next'
+
+import { WalletInfo, CurrentWallet, Utxos, Account } from '../../context/WalletContext'
+// @ts-ignore
+import { useSettings } from '../../context/SettingsContext'
+
+// @ts-ignore
+import Balance from '../../components/Balance'
+// @ts-ignore
+import ToggleSwitch from '../../components/ToggleSwitch'
+import { useBalanceSummary, WalletBalanceSummary } from '../../hooks/BalanceSummary'
+
+import Sprite from './../Sprite'
+import UtxoSelector from './UtxoSelector'
+import AccountSelector from './AccountSelector'
+import LockdateForm, { toYearsRange, lockdateToTimestamp, DEFAULT_MAX_TIMELOCK_YEARS } from './LockdateForm'
+
+import { routes } from '../../constants/routes'
+import * as Api from '../../libs/JmWalletApi'
+
+const timeUtils = (() => {
+  type Milliseconds = number
+  type UnitKey = 'year' | 'month' | 'day' | 'hour' | 'minute' | 'second'
+
+  type Units = {
+    [key in UnitKey]: Milliseconds
+  }
+
+  const units: Units = {
+    year: 24 * 60 * 60 * 1_000 * 365,
+    month: (24 * 60 * 60 * 1_000 * 365) / 12,
```

Comment:
A month is not always (actually never) 365 / 12 days long. A year might not be 365 days long. Does that matter for how this util is used? I'm guessing this is used for rough user feedback only right?

Reply:
> A month is not always (actually never) 365 / 12 days long. A year might not be 365 days long. Does that matter for how this util is used? I'm guessing this is used for rough user feedback only right?

I will add tests for this to make it more clear. Short answer: Should not matter here.
But you are right that I did not take into account what happens on the last day of a month.
Q: Should a "normal" user be able to create a FB that is only valid a few hours, minutes eventually?
It feels like a mistake as the bond will not help much in being chosen more often, but then again, someone might have valid reasons to do so (?)